apps/frontend-pwa/src/pages/course/[courseId]/index.tsx

```
@@ -613,42 +613,57 @@
+    }
+
+    return addApolloState(apolloClient, {
       props: {
-        participantToken,
-        cookiesAvailable,
         courseId: ctx.params.courseId,
         messages: (await import(`@klicker-uzh/i18n/messages/${ctx.locale}`))
           .default,
       },
+    })
+  } catch (error) {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)